python/lsst/atmospec/spectraction.py

```diff
@@ -133,22 +138,38 @@ def dumpParameters():
             if not item.startswith("__"):
                 print(item, getattr(parameters, item))
 
-    def spectractorImageFromLsstExposure(self, exp, *, target_label='', disperser_label='', filter_label=''):
+    def debugPrintTargetCentroidValue(self, image):
+        x, y = image.target_guess
+        self.log.debug(f"Image shape = {image.data.shape}")
+        self.log.debug(f"x, y = {x}, {y}")
+        x = int(np.round(x))
+        y = int(np.round(y))
+        self.log.debug(f"Value at {x}, {y} = {image.data[y, x]}")
+
+    def spectractorImageFromLsstExposure(self, exp, xpos, ypos, *, target_label='',
+                                         disperser_label='', filter_label=''):
         """Construct a Spectractor Image object from LSST objects.
 
         Internally we try to use functions that calculate things and return
         them and set the values using the return rather than modifying the
         object in place where possible. Where this is not possible the methods
         are labeled _setSomething().
         """
+        # TODO: passing exact centroids seems to be causing a serious
```

Comment:
Ticket for this issue?

Reply:
I'm not certain that this is actually still an issue, but I agree it's a scary TODO to have without a ticket, so I'll file one and just mark it as invalid if it turns out to be OK later.